main/provider/helpers.ts

```diff
@@ -69,6 +69,13 @@ export function feeTotalOverMax (rawTx: TransactionData, maxTotalFee: number) {
   return totalFee > maxTotalFee
 }
 
+function parseValue(value?: string) {
+  const parsedHex = value && parseInt(value, 16)
```

Comment:
don't think you need a few of these checks, as `parseInt(undefined)` evaluates to `NaN` and `NaN` is falsy so we dont need an explicit check for it